lib/block-supports/shadow.php

```diff
@@ -51,6 +51,10 @@ function gutenberg_apply_shadow_support( $block_type, $block_attributes ) {
 		return array();
 	}
 
+	if ( wp_should_skip_block_supports_serialization( $block_type, 'shadow' ) ) {
+		return array();
+	}
+
```

Comment:
This bug fix regarding skipping serialization might be something we should split out into its own PR and backport to the WP 6.5 RC.